### PR TITLE
Mathpix.MathpixSnippingTool version 03.00.0074

### PR DESCRIPTION
**Pull request by Bubbu0129**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/56623)

### CODE REVIEW

File: manifests/m/Mathpix/MathpixSnippingTool/03.00.0074/Mathpix.MathpixSnippingTool.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Mathpix.MathpixSnippingTool
+PackageVersion: 03.00.0074
+PackageLocale: en-US
+Publisher: Mathpix
+PackageName: Mathpix Snipping Tool
+License: Copyright (c) Mathpix
```
Comment (Trenly) [inline manifests/m/Mathpix/MathpixSnippingTool/03.00.0074/Mathpix.MathpixSnippingTool.locale.en-US.yaml:9]: A copyright is not a license; Please update this to an appropriate license such as `Proprietary`. `GPL-2.0`, `MIT`, or whatever the actual software license is

File: manifests/m/Mathpix/MathpixSnippingTool/03.00.0074/Mathpix.MathpixSnippingTool.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Mathpix.MathpixSnippingTool
+PackageVersion: 03.00.0074
+PackageLocale: en-US
+Publisher: Mathpix
+PackageName: Mathpix Snipping Tool
+License: Copyright (c) Mathpix
+ShortDescription: Mathpix Snipping Tool Setup
```
Comment (Trenly) [inline manifests/m/Mathpix/MathpixSnippingTool/03.00.0074/Mathpix.MathpixSnippingTool.locale.en-US.yaml:10]: Please provide a better short description of what Mathpix Snipping Tool is

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/m/Mathpix/MathpixSnippingTool/03.00.0074/Mathpix.MathpixSnippingTool.locale.en-US.yaml (hunk 1 above)
Comment (Crisper-Dong) [reply manifests/m/Mathpix/MathpixSnippingTool/03.00.0074/Mathpix.MathpixSnippingTool.locale.en-US.yaml]: Found it on https://github.com/Mathpix/mathpix-markdown-it/blob/master/LICENSE. It's a MIT License.

File: manifests/m/Mathpix/MathpixSnippingTool/03.00.0074/Mathpix.MathpixSnippingTool.locale.en-US.yaml (hunk 2 above)
Comment (Crisper-Dong) [reply manifests/m/Mathpix/MathpixSnippingTool/03.00.0074/Mathpix.MathpixSnippingTool.locale.en-US.yaml]: Done.

File: manifests/m/Mathpix/MathpixSnippingTool/03.00.0074/Mathpix.MathpixSnippingTool.locale.en-US.yaml (hunk 1 above)
Comment (Crisper-Dong) [reply manifests/m/Mathpix/MathpixSnippingTool/03.00.0074/Mathpix.MathpixSnippingTool.locale.en-US.yaml]: Simply cannot find the license. I've check nearly everything.
Does the software NEED a license?

File: manifests/m/Mathpix/MathpixSnippingTool/03.00.0074/Mathpix.MathpixSnippingTool.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Mathpix.MathpixSnippingTool
+PackageVersion: 03.00.0074
+PackageLocale: en-US
+Publisher: Mathpix
+PackageName: Mathpix Snipping Tool
+License:
```
Comment (ItzLevvie) [inline manifests/m/Mathpix/MathpixSnippingTool/03.00.0074/Mathpix.MathpixSnippingTool.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by zachcarp** [comment]
Sorry - was looking at wrong PR.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Missing https://github.com/microsoft/winget-pkgs/pull/56623/files#r841238992

**Review by ItzLevvie — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**